http/task_service.go

```diff
@@ -511,6 +511,14 @@ func decodeGetTasksRequest(ctx context.Context, r *http.Request, orgs influxdb.O
 		req.filter.Limit = influxdb.TaskDefaultPageSize
 	}
 
+	if status := qp.Get("active"); status == "true" {
+		active := string(backend.TaskActive)
+		req.filter.Status = &active
+	} else if status := qp.Get("active"); status == "false" {
+		inactive := string(backend.TaskInactive)
+		req.filter.Status = &inactive
+	}
+
```

Comment:
Is this a query parameter? I am not sure if this is redundant or that it should be changed to `status`?

@jademcgough may be able to provide better feedback than I

Reply:
I've fixed it to be "status" and use the status query parameter directly!